Jenkinsfile

```diff
@@ -0,0 +1,62 @@
+timestamps {
+    node("ubuntu18-agent") {
+        catchError {
+            checkout scm
+            dir_exist= sh (
+		        script: "test -d 'tests' && echo 'Y' || echo 'N' ",
+                returnStdout: true
+            ).trim()
+
+            if (dir_exist == 'N'){
+                currentBuild.result= 'FAILURE'
+                echo "No tests directory found! Exiting."
+                return
+            }
+            // Set BRANCH to specific branch of fledge repo, in case you want to run the tests against specific code
+            // e.g. FOGL-xxxx, main etc.
+            try {
+                stage("Prerequisites"){
+                    sh '''
+                        BRANCH='develop'
+                        ${HOME}/buildFledge.sh ${BRANCH} ${WORKSPACE}
+                    '''
+                }
+            } catch (e) {
+                currentBuild.result = 'SUCCESS'
+                echo "Failed to build Fledge; required to run the tests."
```

Comment:
Why Success on fledge build failure? It should be set as `currentBuild.result = 'FAILURE'`